net-mgmt/nrpe3/src/opnsense/service/templates/OPNsense/Nrpe/nrpe.cfg

```diff
@@ -0,0 +1,27 @@
+{% if helpers.exists('OPNsense.nrpe.general.enabled') and OPNsense.nrpe.general.enabled == '1' %}
+
+log_facility=daemon
+log_file=/var/log/nrpe.log
+debug=0
+pid_file=/var/run/nrpe3/nrpe.pid
+nrpe_user=nagios
+nrpe_group=nagios
+
+server_port={{ OPNsense.nrpe.general.server_port }}
+server_address={{ OPNsense.nrpe.general.server_address }}
+allowed_hosts={{ OPNsense.nrpe.general.allowed_hosts }}
+command_timeout={{ OPNsense.nrpe.general.command_timeout }}
+connection_timeout={{ OPNsense.nrpe.general.connection_timeout }}
+{% if helpers.exists('OPNsense.nrpe.general.dont_blame_nrpe') and OPNsense.nrpe.general.dont_blame_nrpe == '1' %}
```

Comment:
afaik checkbox has always the value 1|0 so you could output it immediately instead of using an if construct if the exists check passes.